schemas/github-pull-request-message.yml

```diff
@@ -0,0 +1,39 @@
+id: http://schemas.taskcluster.net/github/v1/github-pull-request-message.json#
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "GitHub Pull Request Message"
+description: |
+  Message reporting that a GitHub pull request has occurred
+type:         object
+properties:
+  version:    {$const: message-version}
+  organizationName:
+    description: |
+      The GitHub `organizationName` which had an event.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
+  repositoryName:
+    description: |
+      The GitHub `repositoryName` which had an event.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
+  action:
+    description: |
+      The GitHub `action` which triggered an event.
+    type:         string
+    minLength:    {$const: identifier-min-length}
+    maxLength:    {$const: identifier-max-length}
+    pattern:      {$const: identifier-pattern}
+  details:
+    type:         object
+    description: |
+      Metadata describing the pull request.
```

Comment:
I agree with you here. My thinking was, after I finish the worker, I'll strip out all of the fields that the worker doesn't need.